pennylane/_version.py

```diff
@@ -16,4 +16,4 @@
 Version number (major.minor.patch[-label])
 """
 
-__version__ = "0.21.0-dev"
+__version__ = "0.22.0-dev"
```

Comment:
Will there be a separate PR to merged in just before we release and then we merge this one afterwards? Otherwise this should be `__version__ = "0.21.0"` correct?

Reply:
That will be merged into the `v0.21.0-rc0` branch. :slightly_smiling_face: We use that branch once we make the release, instead of `master`.

Reply:
With this approach, we  always have
```python
__version__ = "0.XX.0-dev"
```
on `master`.

Reply:
> That will be merged into the `v0.21.0-rc0` branch. 🙂 We use that branch once we make the release, instead of `master`.

Ahh yes of course!